third_party/jit/assembler.cc

```diff
@@ -2239,15 +2249,25 @@ void Assembler::movq(Register dst, XMMRegister src) {
   emit_sse_operand(src, dst);
 }
 
+void Assembler::movq(const Operand &dst, XMMRegister src) {
+  DCHECK(Enabled(SSE2));
+  EnsureSpace ensure_space(this);
+  emit(0x66);
+  emit_rex_64(src, dst);
+  emit(0x0F);
+  emit(0x7E);
+  emit_sse_operand(src, dst);
+}
+
 void Assembler::movq(XMMRegister dst, XMMRegister src) {
   DCHECK(Enabled(MMX));
   EnsureSpace ensure_space(this);
   if (dst.low_bits() == 4) {
     // Avoid unnecessary SIB byte.
-    emit(0xf3);
+    emit(0xF3);
```

Comment:
Seems there are other occurrences of lower case 'f' and 'e' other places in the code like line 2267 below. Would it makes sense to make this consistent once and for all or do you prefer slowly fixing them as we stumble on them going forward?

Reply:
Thanks for the review. While I agree that it is a little annoying that the hex constants in the assembler are inconsistent w.r.t. case, I am a hesitating to just change them without having a good way of ensuring that we do not introduce new bug. These are notoriously hard to find.